whelktool/scripts/2021/05/lxl-3552-suecia-digital-reproductions/script.groovy

```diff
@@ -0,0 +1,221 @@
+List tif = new File(scriptDir, "suecia-tif.csv").readLines()
+List pdf = new File(scriptDir, "suecia-pdf.csv").readLines()
+
+Map mappings = Collections.synchronizedMap([:])
+List newRecords = Collections.synchronizedList([])
+
+tif.drop(1).each { row ->
+    List cols = row.split(/,(?!.\.tif)/)
```

Comment:
Parsing CSV manually is generally risky business because of the subtleties of the format.
In this case it's fine because the input data is under control.

Reply:
@olovy Agree. Went for a naive implementation this time :)